examples/load_graph_archngv.sbatch

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+#SBATCH --job-name="archngv"
+#SBATCH --nodes=1
+
+#SBATCH --account=proj16
+#SBATCH --partition=prod
+#SBATCH --constraint=cpu
+#SBATCH --time=00:30:00
+
+#SBATCH --cpus-per-task=2
+#SBATCH --exclusive
+#SBATCH --mem=0
+#SBATCH --output="%x-%j.log"
+
+JOB_SCRIPT=$(scontrol show job ${SLURM_JOB_ID} | awk -F= '/Command=/{print $2}')
```

Comment:
Do you need a sbatch script for load_graph.py since it uses only one core and without MPI?

Reply:
This script takes a lot of time.  For the rat, it takes about 30 minutes. For the mouse, it can take many hours. I think it is better to run it on a node on bb5. 
It uses a parallelization based on multiprocessing/joblib, so we can still use several cores.